content/master/cli/command-reference.md

```diff
@@ -609,3 +609,44 @@ URL.
 <!-- vale Crossplane.Spelling = YES -->
 {{< /table >}}
 
+### beta convert
+
+As Crossplane evolves, its APIs and resources may change. To help with the 
+migration to the new APIs and resources, the `crossplane beta convert` command
+converts a Crossplane resource to a new version or kind.
```

Comment:
```suggestion
converts an existing Crossplane resource to a new version or kind.
```

Reply:
This is the only suggestion I didnt use. My reasoning is that the resource does not have to exist somewhere, it can be just a yaml file laying around. But feel free to change if I am getting the meaning wrong

Reply:
works for me, thanks for the clarification